### PR TITLE
Ra pt membership2identity

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Replaces  #844

### CODE REVIEW

File: aspnet/includes/identity/identity_replace_membership.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+> [!NOTE]
+> Since this article was written, the ASP.NET Membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). We strongly recommend updating apps to use the [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) platform rather than the Membership providers featured at the time this article was written. ASP.NET Identity has a number of advantages over the ASP.NET Membership system, including :
+>
+> * More secure.
+> * More preformant.
+> * Improved testability.
+> * Support for two-factor authentication.
+> * More features.
```
Comment (Rick-Anderson) [inline aspnet/includes/identity/identity_replace_membership.md:8]: @StephenMolloy  this is the only place the not will occur. I let the GitHub ChatGPT generate the note and it seemed pretty good. Can you make it better? We should probably give them some motivation to update.

File: aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/users-and-roles-on-the-production-website-cs.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ msc.type: authoredcontent
 
 by [Scott Mitchell](https://twitter.com/ScottOnWriting)
 
+> [!NOTE]
+> Since this document was written, the in-box ASP.Net Membership providers have been superseded by '[ASP.Net Identity](https://learn.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity)'. Please update applications to use the [ASP.Net Identity](https://learn.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity) platform instead of the in-box Membership providers.
```
Comment (Rick-Anderson) [inline aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/users-and-roles-on-the-production-website-cs.md:17]:
```suggestion
[!INCLUDE[](~/includes/identity/identity_replace_membership.md)]
```

File: aspnet/includes/identity/identity_replace_membership.md (hunk 1 above)
Comment (StephenMolloy) [inline aspnet/includes/identity/identity_replace_membership.md:8]: "More features.' feels unneccessary, and "More secure" is a bit amorphous. Instead, I might include "Claims-based Identity" and "Greater interoperability with ASP.Net Core".

```suggestion
> * Better performance
> * Improved extensibility and testability
> * Support for OAuth, OpenID Connect, and two-factor authentication
> * Claims-based Identity support
> * Better interoperability with ASP.Net Core
```

Comment (Rick-Anderson) [reply aspnet/includes/identity/identity_replace_membership.md]: @StephenMolloy I took your suggestion, however
>  "More secure" is a bit amorphous. 

Of the bullets listed, 99.99% won't care as their legacy apps won't add OAuth, don't care about perf, etc. The only thing they'd care about is ***more secure***. Last chance to let me add ***more secure*** back.

**Pull request merged**